### PR TITLE
Display appropriate message in console when resultset not available

### PR DESCRIPTION
**Pull request by kapy2010**
[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1249)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: ui/js/controllers/perf/graphs.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ perf.controller('GraphsCtrl', [
                                        }
                                    }, function(error) {
                                        $scope.tooltipContent.revisionInfoAvailable = false;
-                                       console.log("Failed to get revision: " + error.reason);
+                                       console.log(error.data);
```
Comment (wlach) [inline ui/js/controllers/perf/graphs.js]: Could you change this to "Failed to get revision: " + error.data ?

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by wlach** [comment]
This looks fine, in the future could you prefix the bug in the PR so we don't get the autolander warnings? As I said on irc, this would have been fine as a followup to bug 1187469.

**Pull request merged**